core/launcher/pom.xml

```diff
@@ -92,6 +92,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
                 <configuration>
+                    <outputTimestamp>2023-10-17T10:15:30Z</outputTimestamp>
```

Comment:
normally, one would use the date from the git sha or a 1970 date ...
does this specific timestamp have any meaning?

Reply:
It's today and that's pretty much it :).

Using the date from the git sha is a bit counter productive as that means that it will be different for each git commit, which might not be needed.

I can go with 1970 if you prefer but really we don't really care about this date.